target_chains/ethereum/contracts/forge-test/utils/EntropyTestContracts/EntropyDifferentMagic.t.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: Apache 2
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@pythnetwork/entropy-sdk-solidity/EntropyErrors.sol";
+
+import "../../../contracts/entropy/EntropyGovernance.sol";
+import "../../../contracts/entropy/Entropy.sol";
+
+contract EntropyDifferentMagic is
```

Comment:
Do you really need all of this code here to test the magic number thing? I expect you can remove some parts of it (like the upgradeTo stuff). 

it's generally good practice to minimize the amount of code you have hanging around, as code ends up being a maintenence burden over time.